### ISSUE TITLE
Redirects to update the publishers wheeler -> lbs not working

### ISSUE THREAD

**Issue by adrianoamaral** [IATI/ckanext-iati#369]
The publisher id of a publisher, it used to be wheeler(https://iatiregistry.org/publisher/wheeler) and now it’s lbs (https://iatiregistry.org/publisher/lbs). Usually when I make this change it appears in the Redirects tab, however I am not able to see it there. I did click the Update redirects button. 

Maybe Ckan needs to be restarted?
Usually, I email Swaroop to restart Ckan after I make such a change.

The publisher needs to publish their data this week, but they get an error in AidStream related to invalid publisherid when they try to use the new lbs id. Since I can’t see the wheeler->lbs change in the redirects I was not sure if the issue is in the Registry or in AidStream. Hence, I wanted to check if the publisher id was indeed correctly changed.

**Comment by adrianoamaral** [comment]
Not working when hit "Redirects Update"
<img width="805" alt="Screenshot 2021-08-18 at 14 40 56" src="https://user-images.githubusercontent.com/8742917/129908556-51a7aa75-b95b-41b1-b00a-da3c3f6c5499.png">

**Comment by andreaszenasidi** [comment]
@gtkChop @adrianoamaral  do you have any update on this issue? Many thanks.

**Comment by adrianoamaral** [comment]
Swaroop is working on it! Just answered me….

Adriano Amaral
about.me/adriano_amaral
Enviado via IPhone

**Comment by gtkChop** [comment]
@adrianoamaral @andreaszenasidi 

This is fixed.

The problem is in the activity table.  The database table structure has been changed in the new ckan. Updated the code and works well.

Please let me know if have any questions. Thanks

**Comment by andreaszenasidi** [comment]
@gtkChop looks good! Thank you!

**Comment by andylolz** [comment]
> The database table structure has been changed in the new ckan.

^^ This suggests CKAN has recently been updated. Is that the case? Thanks

**Comment by andreaszenasidi** [comment]
> 
> 
> > The database table structure has been changed in the new ckan.
> 
> ^^ This suggests CKAN has recently been updated. Is that the case? Thanks

@gtkChop @jodiegardiner Could you please address this question. Thanks.

**Comment by jodiegardiner** [comment]
The current CKAN version of the repository is 2.9.3 (3 days ago).  

We currently do point releases this like this pretty quickly after CKAN releases them and once we have integrated them into our stack. This one has been out since May:
https://docs.ckan.org/en/2.9/changelog.html

The changes to the Activity Stream tables came in 2.9.0 and we were patching the new Activity Stream system since then. [This commit](https://github.com/IATI/ckanext-iati/commit/8fc50e0bbb408cc6b41ed2060abf9b30203eb389) removes the need for our patch and was simply adjacent to the 2.9.3 upgrade.


So in short, yes, CKAN has been recently updated but only a point release so mostly bugfiixes and security.
